kafka-connector/src/main/java/com/google/pubsub/kafka/source/CloudPubSubSourceTask.java

```diff
@@ -190,15 +191,17 @@ public void start(Map<String, String> props) {
   }
 
   /**
-   * Attempt to ack all ids in {@link #ackIds}. Acks are best-effort, so if acking fails, messages
```

Comment:
I wouldn't remove this comment. Acks still are best effort since Cloud Pub/Sub has an at-least-once delivery guarantee, so duplicates are still possible.